guide/yaml/custom-entities.md

```diff
@@ -243,6 +243,21 @@ so that the `$message` we passed above gets logged and reported back:
           period: 1s
           command: tail -1 server-input
 
+##### Windows Command Sensor
+
+Like SshCommandSensor blueprint above, this one also uses initializers to define sensors on the entity:
```

Comment:
Like the the blueprint above, the following example also uses `brooklyn.initializers` to define sensors on the entity, this time however it is a windows VM and uses `WinRmCommandSensor`.